Pdf_to_Pdf-A4/Pdf_to_Pdf-A4/Pdf_to_Pdf-A4.csproj

```diff
@@ -9,7 +9,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="SkiaSharp" Version="2.88.0" />
+    <PackageReference Include="SkiaSharp" Version="2.88.8" />
```

Comment:
Can you change to version 2.88.6, because we are using this version at now for our source